llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1239,6 +1239,12 @@ def FeatureXF32Insts : SubtargetFeature<"xf32-insts",
    "v_mfma_f32_16x16x8_xf32 and v_mfma_f32_32x32x4_xf32"
  >;
 
+def FeatureDynamicVGPR : SubtargetFeature <"dynamic-vgpr",
```

Comment:
Where is this target feature enabled?

Reply:
My understanding is graphics front-end adds this target feature as required.  This is similar to `cumode` and `xnack`.

Reply:
That's right, this is enabled from above the backend.